tests/Microsoft.Identity.Test.Integration.net45/SeleniumTests/FociTests.cs

```diff
@@ -49,9 +49,22 @@ public void TestInitialize()
         /// <remarks>The FOCI flag does not appear in the U/P flow, an interactive flow is required. Interactive flow
         /// cannot be automated because http://localhost cannot currently be added to the family apps</remarks>
         [TestMethod]
-        public async Task FociSingInSignOutAsync()
+        public async Task FociSignInSignOutAsync()
         {
             LabResponse labResponse = await LabUserHelper.GetDefaultUserAsync().ConfigureAwait(false);
+            await FociSignInSignOutAsync(labResponse).ConfigureAwait(false);          
+        }
+
+        [TestMethod]
+        public async Task FociSignInSignOutMsaUserAsync()
```

Comment:
Hmm, does this test pass? Foci with client_ids that have MSA passthrough enabled....